service/worker/deployment/deployment_util.go

```diff
@@ -243,3 +244,28 @@ func (d *DeploymentWorkflowClient) escapeChar(s string) string {
 	s = strings.Replace(s, DeploymentWorkflowIDDelimeter, `\`+DeploymentWorkflowIDDelimeter, -1)
 	return s
 }
+
+type DeploymentNameWorkflowClient struct {
```

Comment:
why is this a struct instead of just a function? this isn't a "Client", the client is outside. this is just mapping a deployment name to a workflow id